src/main/webapp/groupContract.jsp

```diff
@@ -0,0 +1,48 @@
+<%-- 
+    Document   : groupContract
+    Created on : Sep 10, 2019, 10:04:53 AM
+    Author     : Malte
+--%>
+
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<!DOCTYPE html>
+<html>
+    <head>
+        <!-- meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Group Contract</title>
+
+        <!-- Insert bootstrap CSS - integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" -->
+        <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+
+        <!-- Custom Stylesheets below this -->
+
+    </head>
+    <body>
+        <!-- Page Content -->
+
+        
+        <!-- OutPut container -->
+        <div class="container">
+            <h1 id="groupContractTitle"></h1>
+            
+            
+            <per id="OutPutReceiver">
+
+            </per>
+
+        </div>
+
+
```

Comment:
Needs a link back to index as per the [assignment](https://docs.google.com/document/d/12hKo2N_VAHISCKVAxdqMpeiKGDSJajBzHJ182bHUdPU/edit#heading=h.oqjxt6ozs8vl):
```
(...), at the bottom of the page, include a link leading back to the index page.
```

Reply:
On it.

Reply:
https://github.com/Runi-VN/CA-1/pull/22/commits/ef30bd68f0f99fe95767887c41634f32ad66623b